packages/blockchain-wallet-v4-frontend/src/data/components/sendBtc/sagas.ts

```diff
@@ -71,6 +72,10 @@ export default ({
       })
       payment = yield payment.init()
       let defaultAccountR
+
+      const account: AccountType = yield call(api.getSBPaymentAccount, 'BTC')
+      yield put(A.sendBtcAccountsSuccess(account))
```

Comment:
I would much rather do all of this in the send/sagas.ts, similar to exchangePaymentAccount

https://github.com/blockchain/blockchain-wallet-v4-frontend/blob/philip/wal-61-buy-ui-clean-up/packages/blockchain-wallet-v4-frontend/src/data/components/send/reducers.ts/#L6-L14

Reply:
2